projects/canopy/src/lib/brand-icon/brand-icons.stories.ts

```diff
@@ -120,21 +103,81 @@ const colours = [
   '--color-poppy-red-dark',
 ];
 
-export const standard = () => ({
-  template: `
-    <lg-swatch-brand-icon [size]="size" [colour]="colour" [specificColour]="specificColour"></lg-swatch-brand-icon>
-  `,
-  props: {
-    size: select('size', sizes, 'xs'),
-    colour: select(
-      'example of applying colour globally',
-      colours,
-      '--color-dandelion-yellow',
-    ),
-    specificColour: select(
-      'example of applying colour specifically to an icon',
-      colours,
-      '--color-super-blue',
-    ),
+export default {
+  title: 'Components/Brand Icon',
+  decorators: [
+    moduleMetadata({
+      declarations: [SwatchBrandIconComponent, LgBrandIconComponent],
+    }),
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: notes,
+      },
+    },
+  },
+  argTypes: {
+    size: {
+      options: sizes,
+      description: 'The size of the icon',
+      table: {
+        type: {
+          summary: sizes,
+        },
+        defaultValue: {
+          summary: 'sm',
+        },
+      },
+      control: {
+        type: 'select',
+      },
+    },
+    colour: {
+      options: colours,
+      description:
+        'The primary colour of icons globally. Can be changed by overriding the `--brand-icon-fill-primary` CSS variable.',
+      table: {
+        defaultValue: {
+          summary: '--color-dandelion-yellow',
+        },
+      },
+      control: {
+        type: 'select',
+      },
+    },
+    specificColour: {
+      options: colours,
+      description: 'The colour of a specific icon, using the `colour` input',
```

Comment:
I think this is a bit confusing. This in my opinion should be called `colour` since that's the name of the input.

While the above should be call `globalColour`.

Reply:
Ah yes I wondered that too, the original story I think had it like this as well, but I've changed it now to how you suggested. I've also improved the description a little.